site/docs/flink.md

```diff
@@ -149,7 +149,7 @@ Table create commands support the most commonly used [flink create clauses](http
 
 * `PARTITION BY (column1, column2, ...)` to configure partitioning, apache flink does not yet support hidden partitioning.
 * `COMMENT 'table document'` to set a table description.
-* `WITH ('key'='value', ...)` to set [table configuration](../configuration) which will be stored in apache iceberg table properties.
+* `WITH ('key'='value', ...)` to set [table configuration](./configuration.md) which will be stored in apache iceberg table properties.
```

Comment:
Okay, it's better to use `./configuration.md` rather than the rendered `../configuration`  ( see the discussion here https://github.com/mkdocs/mkdocs/issues/1649).  Could you pls fix this for other pages (such as spark.md etc) ? Thanks.

Reply:
Ok, I have done. Please check it again?